### ISSUE TITLE
cli: colorize output

### ISSUE THREAD

**Issue by ivotron** [systemslab/popper#541]
Add colors to output to improve what's being shown on the terminal. For this, we can color the square-bracketed prefix that we print currently. For example, this:

```
[my action] docker pull foo
[my action] docker create foo...
[my action] docker start.....
```

would result in having `[my action]` in color. Same for the `--dry-run` prefixes being worked as part of #509 .

**Comment by barbaraDorame** [comment]
I'm interesed in working on this issue.

**Comment by ivotron** [comment]
@barbaraDorame go for it 😃

**Comment by sarthak0906** [comment]
Is this issue still open?
I would like to work on the issue.

**Comment by ivotron** [comment]
@sarthak0906 yes, please go ahead

**Comment by sarthak0906** [comment]
@ivotron I have stuck in a problem.
I did complete the task somewhat from my side but can't run popper to test the output.
can you please help me with it.
Also Is the gitter channel for the project active as it might be easier to get the doubts cleared over there.

**Comment by ivotron** [comment]
yes, Gitter works. You can also describe your problem here, either work.

**Comment by jaimistry12** [comment]
A team of myself and 3 students would like to work on this issue for our UT Austin CS Virtualization class.

**Comment by ivotron** [comment]
hello! thanks a lot for working on this as part of your class 😃  Y'all can start by taking a look at the [contributing guidelines](https://github.com/getpopper/popper/blob/master/CONTRIBUTING.md), setting up a dev environment and running the tests. Please don't hesitate to ask any question you might have about this issue here. We also have a slack workspace that you can join and ask questions there as well.

**Comment by jaimistry12** [comment]
Hello @ivotron thanks for letting us work on this issue! We are at the stage of selecting issues to work on so we will officially start working a bit later down the road. Thanks again! We are in search of 2 more issues to work on so we will comment on a few more soon!

**Comment by athulnair02** [comment]
Hi, is this issue still active? I have experience in colonizing output to the console and can help! Where should I start?
